interface/patient_file/birthday_alert/birthday_pop.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Sharon Cohen<sharonco@matrix.co.il>
+ * @link    http://www.open-emr.org
+ */
+
+
+/**picture free taken from https://pixabay.com/en/balloons-party-celebration-floating-154949*/
```

Comment:
good to go on the picture since using pixabay license (https://pixabay.com/en/service/license/) which is free for any lawful use and no attribution needed.